integration-test/cfn-templates/backbone-demo.yml

```diff
@@ -0,0 +1,100 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Backbone Demo Stack
+Resources:
+
+  IAMUser:
+    Type: AWS::IAM::User
+    Properties:
+      UserName: !Ref "AWS::StackName"
+      Policies:
+        - PolicyName: backbone-core-policy
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "sns:Publish"
+                  - "sns:Subscribe"
+                  - "sns:Unsubscribe"
+                Resource:
+                  - !Ref BackboneCoreTopic
+              - Effect: Allow
+                Action:
+                  - "sqs:CreateQueue"
+                Resource:
+                  - "*"
+              - Effect: Allow
+                Action:
+                  - "sqs:ReceiveMessage"
+                  - "sqs:DeleteMessage"
+                  - "sqs:GetQueueAttributes"
+                  - "sqs:SetQueueAttributes"
+                Resource:
+                  - !Sub "arn:aws:sqs:${AWS::Region}:${AWS::AccountId}:${AWS::StackName}-core-queue"
```

Comment:
it seems to me this isnt defined here.

Reply:
yes because I split the policies into 3 different "PolicyDocuments", so this has the name "backbone-core-policy" which indicates the needed policy when using the backbone-core module. as for this module the queue gets automatically created, we don't have it in cloudformation, so it is just reflecting what would be created in the backbone code.